datalad/distributed/ora_remote.py

```diff
@@ -1057,9 +1057,9 @@ def _verify_config(self, gitdir, fail_noid=True):
     def _get_version_config(self, path):
         """ Get version and config flags from remote file
         """
-
         if self.ria_store_url:
-            target_ri = self.ria_store_url[4:] + path.as_posix()
+            # construct path to ria_layout_version file in store for reporting
+            target_ri = self.ria_store_url[4:] + '/' + path.name
```

Comment:
This is insufficient, since it is not accounting for different locations of the config file. By default there's the `ria-layout-version` at the store level and one per dataset.
What is needed is: Turning `path` into a relative path to the path-part of `self.ria_store_url`. Note, that your assumption that `Path.relative_to` is relatively recent in `pathlib` is wrong, @adswa - you likely looked at `is_relative_to` instead.

Reply:
indeed, that may have been the case. I pushed an alternative, relying on ``self.store_base_path`` - could you check if that approach is better?